src/v/kafka/server/handlers/fetch/replica_selector.cc

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2023 Redpanda Data, Inc.
+ *
+ * Use of this software is governed by the Business Source License
+ * included in the file licenses/BSL.md
+ *
+ * As of the Change Date specified in that file, in accordance with
+ * the Business Source License, use of this software will be governed
+ * by the Apache License, Version 2.0
+ */
+#include "kafka/server/handlers/fetch/replica_selector.h"
```

Comment:
can we put the `.cc` file contents in `fetch.cc` since it is already pretty small? that will help reduce on disk data size from .o files.

Reply:
done